Sources/OpenTelemetryInstrumentationSupport/SpanAttribute+EndUser.swift

```diff
@@ -11,9 +11,15 @@
 //
 //===----------------------------------------------------------------------===//
 
-#if swift(>=5.2)
 import TracingInstrumentation
 
+extension SpanAttributeName {
+    public static let endUserID = "enduser.id"
+    public static let endUserRole = "enduser.role"
+    public static let endUserScope = "enduser.scope"
```

Comment:
I wonder if we should 

/// - See: `EndUserAttributes`

on all of those to at least link like that; I don't want to copy all of the documentation here though, but at least in the docs rendering each attr can point at the right struct to look for full docs.

Reply:
Yep, that's a nice compromise.